audio-worklet/src/bitcrusher-demo.js

```diff
@@ -18,36 +18,59 @@ class BitcrusherDemo {
    * Initalizes and modifies bitcrusher settings in response to GUI input.
    * @param {AudioContext} context the Audio Context
    * @param {String} containerId container for GUI elements
+   * @param {Boolean} useWorkletAndScriptProcessor switch depending on browser
+   *                  which determines if both script processor and audio
+   *                  worklet should be used. If false, only the script
+   *                  processor will be used.
    */
-  constructor(context, containerId) {
+  constructor(context, containerId, useWorkletAndScriptProcessor) {
```

Comment:
I think `useWorkletAndScriptProcessor` is a bit confusing. The script processor is always used, so this really means use a worklet, right?